docs/java-rest/sniffer.asciidoc

```diff
@@ -22,11 +29,17 @@ Add the following to your `pom.xml` file:
 </dependency>
 --------------------------------------------------
 
-The low-level REST client is subject to the same release cycle as
-elasticsearch. Replace `${es.version}` with the desired sniffer version, first
-released with `5.0.0-alpha4`. There  is no relation between the sniffer version
-and the elasticsearch version that the client can communicate with. Sniffer
-supports fetching the nodes list from elasticsearch 2.x and onwards.
+=== Gradle coordinates
+
+Here is how you can configure the dependency using gradle as a dependency manager.
+Add the following to your `build.gradle` file:
+
+["source","groovy",subs="attributes"]
+--------------------------------------------------
+dependencies {
+    compile group: 'org.elasticsearch.client', name: 'sniffer', version: '{version}'
```

Comment:
You can use the short form:
`compile 'org.elasticsearch.client:sniffer:{version}'`